Updated az ad sp createexample

yooakim opened the pull request:
I updated the `az ad sp create---` example with qery to output the variable with names as needed by Packer. Also limited the scope to Contributor.

azurecla commented:
Hi __@saidbleik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bleik). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

bradsev commented:
Accept it. Minor issue corrected. I think Said wanted to get the author attribution. 
Will this checkin get it for him?

pablissima commented:
Brad is listed in the metadata as the author, so it will keep Brad's name as the author on the file.

yooakim committed:
Updated az ad sp createexample

I updated the `az ad sp create---` example with qery to output the variable with names as needed by Packer.
